src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1510,4 +1510,16 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="UseCancellationTokenThrowIfCancellationRequestedDescription" xml:space="preserve">
+    <value>'ThrowIfCancellationRequested' automatically checks whether the token has been canceled, and throws an 'OperationCanceledException' if it has.</value>
+  </data>
+  <data name="UseCancellationTokenThrowIfCancellationRequestedMessage" xml:space="preserve">
+    <value>Use 'ThrowIfCancellationRequested' instead of checking 'IsCancellationRequested'</value>
```

Comment:
Maybe its better to add `throw OperationCanceledException` part too
```suggestion
    <value>Use 'ThrowIfCancellationRequested' instead of checking 'IsCancellationRequested' and throwing 'OperationCanceledException'</value>
```